src/lib/__tests__/utils-test.js

```diff
@@ -1007,202 +1007,89 @@ describe('utils.clearRefinements', () => {
     return helper;
   };
 
-  describe('Without clearsQuery', () => {
```

Comment:
Bunch of tests have been removed because the function (and its signature) has been simplified.